pkg/controller/quayecosystem/constants/constants.go

```diff
@@ -80,6 +80,8 @@ const (
 	DatabaseCredentialsDatabaseKey = "database-name"
 	// DatabaseCredentialsRootPasswordKey represents the key for the database root password
 	DatabaseCredentialsRootPasswordKey = "database-root-password"
+	// DatabaseCredentialsServer represents the key for the database server
+	DatabaseCredentialsServerKey = "server"
```

Comment:
Can the key be called `database-server` to align with the rest of the database parameters?

Reply:
updated to database-server